api/src/main/resources/workbench.yaml

```diff
@@ -1165,6 +1165,25 @@ paths:
           schema:
             $ref: '#/definitions/ErrorResponse'
 
+  /v1/cron/billingProjectBufferStatus:
+    get:
+      security: []
```

Comment:
This makes it so you can call the endpoint unauthenticated. I don't think we want that. It's not particularly sensitive, but I'm not sure I want internal state details to be publicly readable either. Requiring authentication seems like a reasonable roadblock to me.

Reply:
Oh good catch. That's not what I thought that did. 

So if I want regular auth (but not admin-only), then I just drop the `security` and it inherits `aou_oauth`. That's what you meant by inheriting the default?

Reply:
After removing the `security: []` line, I started seeing [this ClassNotFound exception](https://gist.github.com/jaycarlton/c5552ecd77ffca86370a764479f93e9f). Restoring it was the only thing that fixed it. I'm wondering if maybe the security settings aren't at path-level granularity but rather that I need a new controller if I want a different auth setup. It's easy enough to make a new `OpsController` or whatever.

Reply:
Correct.

RE ClassNotFound: That's a result of this nasty piece of code that assumes all controllers live in the API directory: https://github.com/all-of-us/workbench/blob/master/api/src/main/java/org/pmiops/workbench/interceptors/AuthInterceptor.java#L220-L222 , it works for the others here since they are unauthenticated crons (only callable via GAE).

I would just split it into an monitoring/ops controller by changing the tag and putting it in the /api directory - that's probably the easiest fix (though separately we eventually need to improve that auth interceptor code)

Reply:
Yes. I'd write it up but I don't quite understand it. I didn't see any references to directories in the snippet you highlighted itself.

Another benefit of that fix would be to allow controllers to live in packages that are vertically integrated where that makes sense.

Reply:
Ah sorry - I needed to go one level deeper to find the true nastiness I was referring to:
https://github.com/all-of-us/workbench/blob/master/api/src/main/java/org/pmiops/workbench/interceptors/InterceptorUtils.java#L21-L38

You can alternatively add the controller name to this map to have it work:
https://github.com/all-of-us/workbench/blob/master/api/src/main/java/org/pmiops/workbench/interceptors/InterceptorUtils.java#L11-L16

Reply:
OK, that should work. I'm finding several places where naming conventions are mattering in ways I wouldn't expect them to.